### ISSUE TITLE
Gene Ontology not updating

### ISSUE THREAD

**Issue by janelomax** [EBISPOT/OLS#658]
GO in OLS is still on the July release, on the GO website they're on Current release 2022-12-04.

Could you restart the updates please?

**Comment by cmrn-rhi** [comment]
I've noticed the same for some other ontologies, such as FoodOn and HP, for terms released in November 2022. Perhaps all updates need a refresh.

**Comment by Orphanet** [comment]
Same for ORDO
https://www.ebi.ac.uk/ols/ontologies/ordo

bioportal is updated:
https://bioportal.bioontology.org/ontologies/ORDO

thx.

**Comment by serjoshua** [comment]
Good day,

Thank you for raising your concerns. We haven't done any OLS data update for all ontologies this past few months. We have been avoiding data corruption happening in OLS every time we do full updates. But right now, we have been doing it for specific ontologies that needed one (i.e., reports a ticket about an update). As of the moment, we are focused on developing a new version of OLS while trying to support the current version as much as we can. With that said, we have updated GO earlier today to resolve this ticket.

As for FOODON, HP and ORDO, we might try to do an update of all ontologies soon but this is not certain. If not, we will definitely include these ontologies in our next data update. We will keep you posted.

**Comment by cmungall** [comment]
I wasn't aware of this until I heard it on the grapevine just before the holidays. It would be great if you could put up a banner explaining this.

Also this makes it seem like everything is being updated:

<img width="383" alt="image" src="https://user-images.githubusercontent.com/50745/212122697-0220dbd4-825f-404b-99c1-58ed0a2c0d91.png">

**Comment by serjoshua** [comment]
Yes, we tried to do a full release the other day and it failed halfway thru indexing. We might have to continue with targeted releases for now or start doing it in batches. We haven't done any code deployments recently but we might have to consider it to accommodate editing the last updated label and/or displaying a banner.
